hazelcast/src/test/java/com/hazelcast/instance/MobyNamesTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.instance;
+
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static com.hazelcast.util.StringUtil.isNullOrEmptyAfterTrim;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category(QuickTest.class)
+public class MobyNamesTest extends HazelcastTestSupport {
+
+    @Test
+    public void getRandomNameNotEmpty() {
+        String randomName = MobyNames.getRandomName();
+        assertFalse(isNullOrEmptyAfterTrim(randomName));
+    }
+
+    @Test
+    public void getNextRandomNameShouldBeDifferentWithHighProbability() {
```

Comment:
I dont think this test verifies high probability. `assertTrueEventually` checks the condition five times a second. The test requires only one of 50 tries to be different than the initial name.

Reply:
The probability that the next name will be different from the previous is very high because we randomly pick from 99*232 variants. If we count for any of 50 consecutive runs it is even higher. So that's why I called it "WithHighProbability" :) But anyway, I rewrote the test to avoid using assertEventually method.

Reply:
I thought the test was supposed to verify that it is highly probable that a different name is chosen in the second attempt. That is what I was complaining about. Otherwise, the test does not really add much value. Anyway, I'll check the new PR.